src/Frog.cpp

```diff
@@ -53,11 +53,27 @@ sf::FloatRect Frog::GetBounds()
     return frogShape.getGlobalBounds();
 }
 
+sf::Vector2f Frog::GetLocation()
+{
+    return frogShape.getPosition();
+}
+
 void Frog::Catch()
 {
-    frogCatchSound.play();
-    float x = rand() % 760 + 20;
-    float y = rand() % 560 + 20;
-    frogShape.setPosition(x, y);
+    frogCatchSound.play(); 
+}
+
+void Frog::TeleportAwayFromPlayer(sf::Vector2<float> playerLoc)
+{
+    sf::Vector2<float> newFrogLoc{};
```

Comment:
How about
```
sf::Vector2<float> newFrogLoc{};
while(true)
{
    newFrogLoc.x = rand() % 760 + 20;
    newFrogLoc.y = rand() % 560 + 20;

    float distance = VecLength(playerLoc - newFrogLoc);
    if(std::abs(distance) > safeDistance)
        break;
}
frogShape.setPosition(newFrogLoc);
```

You will need to include <cmath> - maybe add that include to DuckyMath, so that once someone includes DuckyMath, they will have all the math they need :D

Also, instead of 760 and 560 magic values, maybe we could use the global WindowWidth and WindowHeight values exposed by Martyna's change regarding keeping player in bounds?